models/feature-flags.go

```diff
@@ -0,0 +1,12 @@
+package models
+
+type FeatureFlag struct {
```

Comment:
I see that you have created a package `models` and then this file. This is fairly common in the java world, not so much in go. 

packages are generally different layers of abstractions. If all you want is to separate models, I'd suggest you create a `models.go` file in the same package where it is being used.

Same thing for other packages too like `utils`. create utils.go file and put all things utils in that file and have it in the same package.

Reply:
You can eventually add packages when there is a need for it later on when there is too much in a file. But I don't think we would need it here tbh.

Reply:
The folder structure which me and @shubham-y were following is like 
```
package githhu.com/Real-Dev-Squad/feature-flag-backend
- package getfeatureflagbyid
- package createfeatureflag
utils.go
```

So I think having a utils file should be there in the root package, so that its being used by all the lambdas.

Reply:
> You can eventually add packages when there is a need for it later on when there is too much in a file. But I don't think we would need it here tbh.

Correct I agree with your point, I've created packages as we will need to change it only at one place, instead of multiple places.

Reply:
Discussed this thing offline.